python/hail/tests/tests_v0_1_cf0e0137a.py

```diff
@@ -860,11 +863,12 @@ def test_concordance(self):
     def test_hadoop_methods(self):
         data = ['foo', 'bar', 'baz']
         data.extend(map(str, range(100)))
+        data = [bytearray(d, encoding="utf8") for d in data]
```

Comment:
I think this is expected in that the hadoop_write method returns a low-level BufferedWriter. If we wanted it to take strings instead of bytes it would have to return a different class and HadoopWriter would have to be string based too. I think wrapping it in TextIOWrapper might do what you need. But it does add line based processing which would be a change in behavior.